HW_4/hw_4_3.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+/*
+Преобразовать Task 2* таким образом, чтобы age принимался через prompt в привязанной вёрстке, а результат выводился в alert
+*/
+
+alert(checkAge_3(prompt("Input your age", 18)));
```

Comment:
Better do it this way
```suggestion
const userValue = prompt("Input your age", 18)
const validationResult = checkAge_3(userValue)
alert(validationResult);
```